.github/workflows/python-package.yml

```diff
@@ -5,17 +5,19 @@ name: Python package
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, feature/** ]
   pull_request:
-    branches: [ main ]
+    branches: [ main, feature/** ]
 
 jobs:
   build:
 
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        # os: [ubuntu-latest, macos-latest, windows-latest]
+        # disabling windows for now
+        os: [ubuntu-latest, macos-latest]
         python-version: [3.7, 3.8, 3.9]
```

Comment:
Surprised `@singledispatchmethod` isn't breaking 3.7

Reply:
I added a python 3.7 workaround for singledispatchmethod here: https://github.com/aws/amazon-braket-default-simulator-python/blob/openqasm-local-simulator/src/braket/default_simulator/openqasm/data_manipulation.py#L640